src/Components/Landing.tsx

```diff
@@ -29,13 +29,13 @@ export const NavBar = (): React.ReactElement => (
         <span className={styles.Love}>Love</span>
       </a>
       <Link to="/subscribe">
-        <span className={styles.NavBarSubscribe}>Subscribe</span>
+        <span className={styles.NavBarSubscribe}><a href="/">Subscribe</a></span>
       </Link>
       <Link to="/blog">
-        <span className={styles.NavBarItems}>Blog</span>
+        <span className={styles.NavBarItems}><a href="/">Blog</a></span>
       </Link>
       <Link to="/">
-        <span className={styles.NavBarItems}>Home</span>
+        <span className={styles.NavBarItems}><a href="/">Home</a></span>
```

Comment:
I removed the `<a>` tags in this PR https://github.com/meganyin13/samwise-positivity/pull/50 since it was giving console errors. We shouldn't need them since `<Link>` already wraps the element in an `<a>` tag.

Reply:
yeah youre right, I didnt realize that

Reply:
the diff states the inner `<a>` tags haven't been removed and starting the application still shows these errors in the developer console
![image](https://user-images.githubusercontent.com/20008134/86424851-a0216d80-bcb1-11ea-862c-d8ca15befe2e.png)

Reply:
hmm not sure why I didn't get this error but no worries I have removed them :)